devsetup/Makefile

```diff
@@ -47,6 +47,7 @@ BMAAS_REDFISH_USERNAME ?= admin
 BMAAS_REDFISH_PASSWORD ?= password
 BMAAS_SUSHY_EMULATOR_NAMESPACE ?= sushy-emulator
 BMAAS_LIBVIRT_USER ?= sushyemu
+BMAAS_ROUTE_LIBVIRT_NETWORKS ?= ${BMAAS_NETWORK_NAME},crc,default
 
```

Comment:
Should we de-dup this list as we use BMAAS_NETWORK_NAME=default for baremetal provisioning?

Reply:
We do, in CI? How does that work, the libvirt default network is DHCP enabled - don't we get a conflict with the Metal3 DHCP service?

Reply:
I will add a follow up to de-duplicate the list of networks to route in `scripts/libvirt-network-routing.sh`.